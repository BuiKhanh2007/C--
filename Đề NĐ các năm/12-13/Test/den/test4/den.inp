109 191
30 230
143 307
36 45
125 145
190 418
160 325
196 212
194 409
153 393
117 289
191 328
115 137
83 127
32 83
130 273
121 374
102 227
189 365
