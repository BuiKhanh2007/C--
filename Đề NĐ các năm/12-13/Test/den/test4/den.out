960
2326
2297
93
244
3335
2341
247
3188
3444
2311
2006
258
504
503
1916
3514
1550
2581
